extensions/amp-story/1.0/page-advancement.js

```diff
@@ -546,16 +546,16 @@ export class ManualAdvancement extends AdvancementConfig {
   }
 
   /**
-   * Checks if click was inside of one of the side edges of the screen.
+   * Checks if click was inside of one of the side edges of the story page.
    * @param {!Event} event
-   * @param {!ClientRect} pageRect
    * @return {boolean}
    * @private
    */
-  isInScreenSideEdge_(event, pageRect) {
+  isInScreenSideEdge_(event) {
+    const storyRect = this.element_.getBoundingClientRect();
```

Comment:
```suggestion
    const pageRect = this.element_.getLayoutBox();
```

Since `element.getBoundingClientRect()` is expensive performance-wise, we prefer to use `getLayoutBox()` which is a cached version of the former. Generally we want to reduce the amount of performance-heavy operations, especially in critical paths like navigation.

Reply:
Ah I changed the way I did this in the next commit. Closing this comment as outdated.